spec/requests/portfolio_items_spec.rb

```diff
@@ -38,9 +38,12 @@
         delete "/api/v0.0/portfolio_items/#{portfolio_item_id}", :headers => admin_headers, :params => valid_attributes
       end
 
-      it 'deletes the record' do
+      it 'discards the record' do
         expect(response).to have_http_status(204)
       end
+      it 'marks the icon as discarded instead of deleting it from the db' do
```

Comment:
minor but leave a blank line between tests

Reply:
Minor and easy to fix!